src/pre_defined_classes/mathematical_program/medium/q4/BasketballGames.java

```diff
@@ -0,0 +1,27 @@
+package pre_defined_classes.mathematical_program.medium.q4;
+
+/*
+You are a scorekeeper for your high school basketball team. It’s the end of the season and you have to calculate your
+team’s participation in all of the tournaments. You decide to do this using a Java program. Your job is to get the
+program to calculate the total score across all the games, find the highest score, and find the lowest score. Here is
+the game statistics:
+            Game #1 - Your school lost (Your coach recorded the score as -36)
```

Comment:
tbh this question does not seem easy to understand - we should bring this up next meeting

Reply:
Ok I will do that

Reply:
Change this to get rid of the win/loss, negative scores, and absolute values